Automattic/pocket-casts-ios
# GRDB: Podcast Data Manager

This is another PR that adds a GRDB layer for `PodcastDataManager`. As the previous one, this works by duplicating and adapting the code.

We're having discussions in the initial PR about having a more generic layer that can work with both. I'm pushing this PR just so we can test GRDB.

## To test

1. Run the app
2. Open a few podcasts
3. ✅ Ensure everything works as usual

## Checklist
